api/src/modules/repo/repo.service.ts

```diff
@@ -16,12 +16,12 @@ export interface IRepoGitProvider {
   getAllOrgWithPagination(): Promise<Organization[]>;
   getAllRepoOfAllOrgWithPagination(): Promise<Repo[]>;
   getAllRepoOfUserWithPagination(): Promise<Repo[]>;
-  getCommitsOfAllRepoOfAllOrgWithPagination(date: Date): Promise<void>;
-  getCommitsOfAllRepoOfUserWithPagination(date: Date): Promise<void>;
-  getIssuesOfAllRepoOfAllOrgWithPagination(): Promise<void>;
-  getIssuesOfAllRepoOfUserWithPagination(): Promise<void>;
-  getPullRequestsOfAllRepoOfAllOrgWithPagination(): Promise<void>;
-  getPullRequestsOfAllRepoOfUserWithPagination(): Promise<void>;
+  getCommitsOfAllRepoOfAllOrgWithPagination(date: Date): Promise<Repo[]>;
```

Comment:
To me, it would have made more sense for this function to return something like `Commit[]`, but I understand why this would be difficult
Why did I expect `[]Commit` ? Because it would mean that the `getCommitsOfAllRepoOfAllOrgWithPagination` have no idea about the entity schema and liaison between repo and commits. Its only job is to return all the commits of all the repos.